doc/nrf/device_guides/working_with_nrf/nrf70/developing/raw_tx_operation.rst

```diff
@@ -0,0 +1,117 @@
+
+.. _ug_nrf70_developing_enabling_raw_packet_transmit_feature:
+
+Enabling raw packet transmit feature
+************************************
+
+Applications that intend to use the raw packet transmit feature need to enable the ``CONFIG_NRF700X_RAW_DATA_TX`` Kconfig option in the project configuration.
+
+.. _ug_nrf70_developing_tx_injection_mode:
+
+TX injection mode
+*****************
+
+The nRF70 Series device supports raw 802.11 packet transmission when TX injection mode is enabled in the nRF Wi-Fi driver. A net_mgmt run-time API ``NET_REQUEST_WIFI_MODE`` has been provided to enable TX injection mode of operation. The same API can be used to disable TX injection mode in the driver when raw 802.11 packet transmission is not required any more by the application.
+
+TX injection mode can be enabled when the nRF70 Series device operates in Station mode, both when it is operating as connected Station or as an unconnected Station (i.e. not connected to Access Point).
+
+When TX injection mode is enabled, the user needs to also configure the operating channel. This channel will be used as the default transmit channel when the nRF70 device operates as a **non-connected** Station. When the device operates as a  Station **connected** to an AP on a given channel, the same channel will be used for raw 802.11 packet transmission mode.
+
+The net_mgmt API ``NET_REQUEST_WIFI_CHANNEL`` can be used to set the desired channel for raw 802.11 packet transmission.
+
+See the :ref:`wifi_shell_sample` sample for more information on configuring mode setting and channel setting for raw packet transmission via shell commands.
+
+The shell commands and network management APIs to switch between operational modes and to set channel are provided below
+
+.. list-table:: Wi-Fi raw packet transmission network management APIs
+   :header-rows: 1
+
+   * - Network management APIs
+     - Wi-Fi shell command
+     - Example Usage
+     - Description
+   * - net_mgmt(NET_REQUEST_WIFI_MODE)
+     - wifi mode -i<interface instance> <configuration>
+     - wifi mode -i1 -st
+     - configure interface instance 1 to Station + TX injection mode
+   * - net_mgmt(NET_REQUEST_WIFI_MODE)
+     - wifi mode -i<interface instance> <configuration>
+     - wifi mode -i1 -s
+     - configure interface instance 1 to pure station mode
+   * - net_mgmt(NET_REQUEST_WIFI_CHANNEL)
+     - wifi channel -i<interface instance> <channel number to set>
+     - wifi channel -i1 -c6
+     - set raw transmission channel to 6 for interface 1 to be used in Station idle mode
+
+.. _ug_nrf70_developing_tx_injection_mode_usage_requirements:
+
+TX injection mode usage requirements
+************************************
+
+The raw packet to be transmitted by an application should be a fully conformant IEEE 802.11 packet as defined below.
```

Comment:
Double-check on this: is it a "should" or a "must"? 
Assuming that it is a must, perhaps worth adding also a "note" to tell users that non-conformant packets will be checked and dropped at the nRF70 device and won't be transmitted on - air.

Reply:
done